app/models/user.rb

```diff
@@ -35,6 +35,7 @@ class User < ApplicationRecord
   include DeviseTokenAuth::Concerns::User
 
   enum gender: { female: 0, male: 1, fluid: 2 }
-  has_many :targets, dependent: :destroy
   validates :gender, presence: true
+
+  has_many :targets, dependent: :destroy
```

Comment:
Please add a test for this

Reply:
This test was added here https://github.com/LeandroSoria2/TargetApi/blob/develop/spec/models/user_spec.rb#L41